examples/plot_kmeans_manifolds.py

```diff
@@ -44,7 +44,7 @@ def kmean_poincare_ball():
         space="H2_poincare_disk",
         marker=".",
         color="black",
-        point_type=manifold.point_type,
+        point_type=manifold.coords_type,
```

Comment:
Why the change of name?

point_type is a purely computational attribute, but coords_type often refers to why coordinate system can be used on a manifold, which is a mathematical attribute.

Reply:
Nevermind, it seems that point_type was wrongfully used here: it is indeed a coords_type